### DISCUSSION TITLE
New Version Version: 2023.3.0 Unable to import the csv of keepassX, there is no response after clicking.

### DISCUSSION THREAD

**Discussion by inetclouds** [dani-garcia/vaultwarden#3647] [General]
New Version Version: 2023.3.0 Unable to import the csv of keepassX, there is no response after clicking.

**Comment by av3** [comment]
Same behaviour when trying to import csv of Roboform

**Reply by stefan0xC** [reply]
I think this has been fixed upstream in [web-v2023.4.0](https://github.com/bitwarden/clients/releases/tag/web-v2023.4.0) see https://github.com/bitwarden/clients/pull/5137

**Comment by inetclouds** [comment]
ok

**Reply by BlackDex** [reply]
Try the `:testing` tagged images, they have `v2023.5.0`

**Reply by inetclouds** [reply]
ok thank you

**Reply by Stitch10925** [reply]
The problem still persists on the v2023.5.0 testing version

**Reply by inetclouds** [reply]
View the author's test version and hope it can be imported, docker pullVaultwarden/server:testing

**Reply by inetclouds** [reply]
:testing Is this version ready for official use?

**Comment by inetclouds** [comment]
View the author's test version and hope it can be imported, docker pullVaultwarden/server:testing

**Comment by inetclouds** [comment]
At present, the latest version of docker is version 3.0 3 months ago, but I found that the test version with the author's label is 5.0. I tried to install it yesterday and found that the problem was solved. I don't know when the author will release the official version. Because it is a test version label, I dare not use it in a formal environment. Looking forward to the author's official release.

**Comment by inetclouds** [comment]
I have been using keepass before, but I just started using this one. I don’t know how to manually upgrade to version 5.0 from version 3.0. I thought about directly overwriting the web directory and found that it couldn’t be loaded, so I look forward to the official update of the latest version 5.0 by the author.

**Comment by Stitch10925** [comment]
What's the difference between the "vaultwarden/web-vault" and "vaultwarden/server" images? I am currently using the "vaultwarden/server" test image.

**Reply by BlackDex** [reply]
the `web-vault` is only the web-vault code. That container is used by `server` during the container build.
This way we can control the web-vault code in a safe controlled way in that we know which hash is used during the build of the `server` containers. That hash is also visible in the GitHub Actions logs of the `bw_web_vault` repo so that everybody can match those items if they want to know it's build using safe code.
